sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentClassifierAdministrationLiveTests.cs

```diff
@@ -134,12 +134,103 @@ public async Task BuildDocumentClassifierWithAzureBlobFileListSource()
 
         #endregion Build
 
-        #region Delete
+        #region Get
+
+        [RecordedTest]
+        [TestCase(true, Ignore = "https://github.com/Azure/azure-sdk-for-net/issues/35243")]
```

Comment:
Service is returning 400 for this particular test when using AAD, for some reason.